internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -1713,6 +1721,10 @@ func resourceVirtualMachineScaleSetExtensionHash(v interface{}) int {
 			buf.WriteString(fmt.Sprintf("%t-", v.(bool)))
 		}
 
+		if v, ok := m["automatic_upgrade_enabled"]; ok {
+			buf.WriteString(fmt.Sprintf("%t-", v.(bool)))
+		}
+
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```